tutorials/primitives/01_introduction_to_primitives.ipynb

```diff
@@ -0,0 +1,936 @@
+{
```

Comment:
<p>I like this diagram, but it feels a bit weird having the start box on the right hand side and then reading right to left. Can you invert the diagram so that the user is reading from left to right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1385/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
How does this look? @javabster @ikkoham ![FlippyflipQRT](https://user-images.githubusercontent.com/109966025/223411818-2731c61b-387a-4d87-82a6-fe2e2e1cd6b1.png)

Reply:
How does this look @javabster @ikkoham? 😁 ![FlippyflipQRT](https://user-images.githubusercontent.com/109966025/223412290-bb21a7f1-d099-4954-8d49-581c8b4c2cf7.png)

Reply:
I think this looks better going from left to right but I agree with steve the the Yes/No is maybe a bit too aggressive with the colours here, perhaps something more neutral or not coloured box at all would be better

Reply:
How does this look? 😁 @javabster @woodsp-ibm 

![Light Blue Modern Business Twitter Header (56)](https://user-images.githubusercontent.com/109966025/223613387-d09b0037-3520-471b-9f07-78ebc4d643fe.png)

Reply:
Nice! Please push this to the PR and let's merge this PR.